main_Script/main_Script.ino

```diff
@@ -137,16 +129,17 @@ void loop()
   {
     deltaThetaRadFine1 = ((pixy.blocks[0].x)*convertPixToDegFine - centerOffsetDegFine)*convertDegToRad;
     deltaThetaRad = deltaThetaRadFine1;
-
+    fsInjection(&deltaThetaRad, fmState);
```

Comment:
We should pass in `deltaThetaRadFine1`. This is because, once the system is complete, we will have `deltaThetaRadFine1` and `deltaThetaRadFine2`, and conditionally assign one of them to `deltaThetaRad` based on the state of `fmState->activeFS`, and we only want to fault the primary fine sensor